test/compare/output.js

```diff
@@ -4,13 +4,11 @@
 }())
 
 // should add if missing
-var x = 2
-;[1, 2, 3].map(function() {})
+var x = 2;[1, 2, 3].map(function() {})
 // comment
 ;[7, 8].forEach(doStuff)
 
-var y = 8
-;(function() {
```

Comment:
uhm.. weird that this behavior changed. I would not expect the code to be formatted this way.